qct_parse/qct_parse.py

```diff
@@ -21,6 +21,9 @@
 import shutil 			# dependency checking
 
 
+CONFIG_ENVIRONMENT_VARIABLE_NAME = 'QCT_PARSE_CONFIG_DIRECTORY'
```

Comment:
is this a place holder? Or does this connect to something in the pyproject.toml?